src/attrs_strict/__init__.pyi

```diff
@@ -1,8 +1,6 @@
-from ._error import (
-    AttributeTypeError as AttributeTypeError,
-    BadTypeError as BadTypeError,
-    TupleError as TupleError,
-    TypeValidationError as TypeValidationError,
-    UnionError as UnionError,
-)
+from ._error import AttributeTypeError as AttributeTypeError
```

Comment:
is there some pep that recommends this ?

Reply:
nope this was black/isort work